app/vmalert/datasource/datasource.go

```diff
@@ -1,14 +1,14 @@
 package datasource
 
-import "context"
-
-// Metrics the data returns from storage
-type Metrics struct{}
-
-// VMStorage represents vmstorage entity with ability to read and write metrics
-type VMStorage struct{}
+// Metric represents single metric
```

Comment:
the description is not clear - which metric, how we get it and what we do with it?